andela/strikers-ah-backend
# #164489773 setup eslint

#### What does this PR do?
1. setup eslint : eslint, eslint-config-airbnb, eslint-plugin-import, eslint-plugin-jsx-a11y, eslint-plugin-react
#### Description of Task to be completed?
Have eslint in our project.
#### How should this be manually tested?
1. check `package.json` under `devdependencies`.
2. when cloning or pulling this branch do `npm install`
#### What are the relevant pivotal tracker stories?
Configure ESLint #164489773
[Pivotal Link](https://www.pivotaltracker.com/story/show/164489773)
#### Screenshots (if appropriate)
<img width="432" alt="Screen Shot 2019-03-19 at 12 00 41" src="https://user-images.githubusercontent.com/37753967/54597127-b06ef400-4a3e-11e9-9ffd-87bc42e4b04b.png">
